sysobject.ids

```diff
@@ -6348,8 +6353,11 @@
 18334.1.1.1.2.1.83.3.1	Konica	PRINTER	bizhub C364
 18334.1.1.1.2.1.84.2.1	Konica	PRINTER	bizhub C284
 18334.1.1.1.2.1.84.3.1	Konica	PRINTER	bizhub C284
+18334.1.1.1.2.1.87.2.1	Konica	PRINTER	bizhub 36
 18334.1.1.1.2.1.103.2.1	Konica	PRINTER	bizhub C554
 18334.1.1.1.2.1.104.2.1	Konica	PRINTER	bizhub C454E
+18334.1.1.1.2.1.110.3.4	Konica	PRINTER	bizhub 364E
```

Comment:
Thank you @ddurieux for the PR
Can you just keep OID as ordered as possible ? `18334.1.1.1.2.1.110.3.4 ...` should appear just after `18334.1.1.1.2.1.107.3.1`